sample-queries/sample-queries.json

```diff
@@ -1981,6 +1981,80 @@
         "postBody": "{\r\n  \"applicationName\": \"LinkedResource created from Microsoft Graph Explorer\"\r\n}",
         "tip": "This query requires the Tasks.ReadWrite permission. To find a value for the taskListId parameter, you can run: GET https://graph.microsoft.com/beta/me/todo/lists. to find a value for the taskId, you can run: GET https://graph.microsoft.com/beta/me/todo/lists/{taskListId}",
         "skipTest": false
+    },
+    {
+        "id": "fd58d371-315e-49d8-aa74-58fc4ad1337c",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "list Azure AD gallery apps",
+        "requestUrl": "/beta/applicationTemplates",
+        "docLink": "https://docs.microsoft.com/graph/api/applicationtemplate-list?view=graph-rest-beta",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/appTemplateAPISurvey",
+        "skipTest": false
+    },
+    {
+        "id": "101af4d8-4697-4347-906e-9c3659bf2a29",
+        "category": "Identity and Access",
+        "method": "POST",
+        "humanName": "instantiate an Azure AD gallery app",
+        "requestUrl": "/beta/applicationTemplates/{id}/instantiate",
+        "docLink": "https://docs.microsoft.com/graph/api/applicationtemplate-instantiate?view=graph-rest-beta",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/appTemplateAPISurvey",
+        "postBody": "{\r\n \"displayName\": \"appDisplayName\"\r\n}",
+        "skipTest": false
+    },
+    {
+        "id": "3155cff4-2d16-4d45-a9f7-214b6a246dc5",
+        "category": "Identity and Access",
+        "method": "PATCH",
```

Comment:
We can add a sample request `postBody` with just a single property as supplied in the [docs](https://docs.microsoft.com/en-us/graph/api/serviceprincipal-update?view=graph-rest-1.0&tabs=http#request); to help our users with the initial request construction.

Reply:
Added.

Reply:
Added the postBody